### PR TITLE
Fixing breakage when 3scale is not used

### PR DESCRIPTION
**Pull request by secondsun**

### CODE REVIEW

File: src/app/main.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ const bootstrap = function() {
             }
           });
         }
-        if (syncConfig.headers) {
+        if (syncConfig.headers && syncConfig.headers.appKey) {
```
Comment (aidenkeating) [inline src/app/main.ts]: @secondsun Going to double check whether this is `appKey` or `app_key`

Comment (aidenkeating) [reply src/app/main.ts]: @secondsun Yep we should probably check for `syncConfig.headers.app_id` and `syncConfig.headers.app_key`

**Comment by secondsun** [comment]
@aidenkeating Fixed

**Comment by secondsun** [comment]
@aidenkeating I can make your computer beep!

**Pull request merged**